src/ert/plugins/hook_implementations/workflows/__init__.py

```diff
@@ -1,14 +1,12 @@
-from ert.shared.plugins.plugin_manager import hook_implementation
-from ert.shared.plugins.plugin_response import plugin_response
+import ert
 
 from .disable_parameters import DisableParametersUpdate
 from .export_misfit_data import ExportMisfitDataJob
 from .export_runpath import ExportRunpathJob
 from .misfit_preprocessor import MisfitPreprocessor
 
 
-@hook_implementation
-@plugin_response(plugin_name="ert")  # type: ignore
+@ert.plugin(name="ert")  # type: ignore
```

Comment:
Is the type:ignore still needed here?

Reply:
Yes, mypy will error when you have a redundant `# type: ignore`.